ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -321,3 +322,43 @@ def install_ocs_ui(self):
         self.install_ocs_operator()
         self.verify_ocs_operator_succeeded()
         self.install_storage_cluster()
+
+    def verify_channel(self, timeout_upgrade=600, channel="4.7"):
+        """
+        Verify Channel
+
+        timeout_install (int): Time in seconds to wait
+        channel (str): ocs y version
+
+        """
+        self.navigate_installed_operators_page()
+        self.do_click(self.dep_loc["ocs_operator_installed"])
+        self.do_click(self.dep_loc["subscription"])
+        sample = TimeoutSampler(
+            timeout=timeout_upgrade,
+            sleep=20,
+            func=self.check_element_text,
+            expected_text=f"stable-{channel}",
+        )
+        if not sample.wait_for_func_status(result=True):
+            logger.error(
+                f"OCS upgrade status is not Succeeded after {timeout_upgrade} seconds"
+            )
+            raise TimeoutExpiredError
+
+    def upgrade_ocs_ui(self, channel=None):
+        """
+        Upgrade OCS version via UI
+
+        channel (str): ocs y version
+        """
+        self.navigate_installed_operators_page()
+        self.do_click(self.dep_loc["ocs_operator_installed"])
+        self.do_click(self.dep_loc["subscription"])
+        ocs_version = get_ocs_version()
+        if ocs_version == "4.6" and channel == "4.7":
+            self.do_click(self.dep_loc["channel_4_6"])
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1843045 hasn't been fixed, so this test will be rather unstable unless you expect a possibility that the channel change button will not be there in the UI and refresh the page in this case.